pyalgotrading/algobulls/api.py

```diff
@@ -315,18 +317,26 @@ def start_strategy_algotrading(self, strategy_code: str, start_timestamp: dt, en
             }
             _timestamp_format = "%d-%m-%YT%H:%MZ"
             execute_config = {
-                map_trading_type_to_date_key[trading_type]: [start_timestamp.astimezone().astimezone(timezone.utc).isoformat(), end_timestamp.astimezone().astimezone(timezone.utc).isoformat()],
-                'isLiveDataTestMode': trading_type == TradingType.PAPERTRADING,
-                'customizationsQuantity': lots
+                map_trading_type_to_date_key[trading_type]: [start_timestamp.astimezone(timezone.utc).isoformat(), end_timestamp.astimezone(timezone.utc).isoformat()],
+                'isLiveDataTestMode': trading_type in [TradingType.PAPERTRADING, TradingType.REALTRADING],
+                'customizationsQuantity': lots,
+                'brokingDetails': broker_details
             }
+
+            params = None
+            if trading_type in [TradingType.PAPERTRADING, TradingType.BACKTESTING]:
+                endpoint = 'v4/portfolio/strategies?isPythonBuild=true&isLive=false'
+                execute_config['initialFundsVirtual'] = initial_funds_virtual
+            elif trading_type is TradingType.REALTRADING:
+                endpoint = 'v4/portfolio/strategies?isPythonBuild=true&isLive=true'
+                execute_config['brokerId'] = broker_id
```

Comment:
`broker_id` parameter not required as everything is coming through `broking_details`. Please clean everywhere.